v23.1/transaction-retry-error-reference.md

```diff
@@ -1,54 +1,92 @@
 ---
 title: Transaction Retry Error Reference
-summary: A list of the transaction retry (serialization) errors emitted by CockroachDB, including likely causes and user actions for mitigation.
+summary: A list of the transaction retry errors emitted by CockroachDB, including likely causes and user actions for mitigation.
 toc: true
 docs_area: reference.transaction_retry_error_reference
 ---
 
-This page has a list of the transaction retry error codes emitted by CockroachDB.
+When a [transaction](transactions.html) is unable to complete due to another concurrent or recent transaction attempting to write to the same data, CockroachDB will [automatically attempt to retry the failed transaction](transactions.html#automatic-retries) without involving the client (i.e., silently). If the automatic retry is not possible or fails, a _transaction retry error_ is emitted to the client. Multiple transactions trying to write to the same data is also known as contention.
```

Comment:
Same point as I raised on the other file about maybe referring to a glossary definition of contention in order to consolidate, and also provide a bit more info with the canonical definition.